docs/howto/installing-local-extensions.md

```diff
@@ -10,7 +10,7 @@ extensions site.
 
 There are a couple of caveats:
 
-- Must be located in `{web_root}/extensions/local/{author_name}/{extension_name}/`
```

Comment:
That is incorrect based on the rest of the document. In hindsight with the changes in 3.3 `{site_root}` is a better choice, but there is also the directories `{site_root}/public/extensions/ …` where the `public/` part if far more appropriate to call "web root" :smile: